schedule/jeos/sle/jeos-base+phub.yaml

```diff
@@ -0,0 +1,36 @@
+---
+description: 'Test suite verifies functionality of packages present in Package Hub (phub) module'
+name: 'jeos-base+phub'
+conditional_schedule:
+    bootloader:
+        MACHINE:
+            svirt-xen-pv:
+                - installation/bootloader_svirt
+            svirt-xen-hvm:
+                - installation/bootloader_svirt
+                - installation/bootloader_uefi
+            svirt-hyperv-uefi:
+                - installation/bootloader_hyperv
+                - installation/bootloader_uefi
+            svirt-hyperv:
+                - installation/bootloader_hyperv
+                - installation/bootloader_uefi
```

Comment:
Why is the `installation/bootloader_uefi` scheduled for both `svirt-hyperv` and `svirt-hyperv-uefi`?

Reply:
We need to set [the resolution](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/bootloader_setup.pm#L403) in GRUB2. Instead of creating an additional module, `bootloader_uefi` does the job in both cases.

Reply:
have you thought about calling the code of bootloader_svirt and bootloader_hyperv in installation/bootloader? The approach for installation/bootloader was to be the "generic" module that calls the right specific implementations where necessary

Reply:
> have you thought about calling the code of bootloader_svirt and bootloader_hyperv in installation/bootloader? The approach for installation/bootloader was to be the "generic" module that calls the right specific implementations where necessary

Forgot about that module, but anyway we need to interact with grub2 only in case of non kvm workers. So for KVM+virtio we can skip it.

Reply:
yes, this is why it's called installation/bootloader not installation/grub2 :) So you could easily extend it to not do much or anything in case of KVM+virtio

Reply:
> yes, this is why it's called installation/bootloader not installation/grub2 :) So you could easily extend it to not do much or anything in case of KVM+virtio

Alright :D, I will create a ticket for it. Thanks!

Reply:
> bootloader

I thought it was `bootloader_start` the generic module